common/db.go

```diff
@@ -269,14 +273,43 @@ func InitDB(dbPath string) (*DB, error) {
 	d.withdrawableUnbondingLocks = stmt
 
 	// Winning tickets prepared statements
-	stmt, err = db.Prepare("INSERT INTO winningTickets(sender, recipient, faceValue, winProb, senderNonce, recipientRand, recipientRandHash, sig, sessionID) VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)")
+	stmt, err = db.Prepare(`
+	INSERT INTO winningTickets(createdAt, sender, recipient, faceValue, winProb, senderNonce, recipientRand, recipientRandHash, sig, creationRound, creationRoundBlockHash)
+	VALUES(datetime('now'), :sender, :recipient, :faceValue, :winProb, :senderNonce, :recipientRand, :recipientRandHash, :sig, :creationRound, :creationRoundBlockHash)
```

Comment:
Since `createdAt` has the `DEFAULT CURRENT_TIMESTAMP` property, do we need to explicitly insert it as `datetime('now')` ?

Reply:
Ah yes 👍  this must have snuck back in during a rebase